ownership-chain/runtime/laos/src/configs/fee.rs

```diff
@@ -0,0 +1,201 @@
+use crate::PhantomData;
```

Comment:
I think this should be either part of `evm` or `transaction_payment` or more external module `types.rs` (if it is used more commonly), because it is not a pallet `fee` configuration

Reply:
The naming as `configs\` rather than `pallet_configs\` is intentional. The software is employed by various pallets, suggesting its application is broader than a single pallet's scope. Although integrating it into `evm`, `transaction_payment`, or a more universal module like `types.rs` was considered, especially since it's not strictly a pallet fee configuration, I opted for `fee.rs` to ensure `mod.rs` does not become cluttered. 
This choice aligns with the aim of maintaining clear and organized module structures.

Reply:
> The naming as configs\ rather than pallet_configs\ is intentional
I still don't see anything about configuration here. It's a struct that is passed to configuration of other pallets, it's not a configuration by itself

> I opted for fee.rs to ensure mod.rs does not become cluttered.
I am not suggesting to move it to `mod.rs`. I meant `types.rs`, which could be `runtime/laos/src/types.rs` or `impls.rs`, similar to [this](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/common/src/impls.rs)

Reply:
and in general, this brings a practice that would force us having an entire file for any auxillary type/struct like this, which is not really sustainable, IMO

Reply:
I still don't understand how "moving things into configs" affects this, it's simply renaming and moving the file outside of `configs/`